controls/e8.yml

```diff
@@ -0,0 +1,177 @@
+id: 'e8'
+policy: 'e8'
+source: 'https://www.cyber.gov.au/sites/default/files/2023-11/PROTECT%20-%20Hardening%20Linux%20Workstations%20and%20Servers%20%28November%202023%29.pdf'
+title: 'Australian Cyber Security Centre (ACSC)'
+levels:
+    - id: base
+
+controls:
+    -   id: 'patching'
+        levels:
+            - base
+        title: 'Application and operating system patching'
+        rules:
+            - package_talk_removed
+            - package_talk-server_removed
+            - package_ypbind_removed
+            - package_telnet_removed
+            - service_telnet_disabled
+            - package_telnet-server_removed
+            - package_rsh_removed
+            - package_rsh-server_removed
+            - package_quagga_removed
+            - service_avahi-daemon_disabled
+            - package_squid_removed
+            - service_squid_disabled
+            - ensure_redhat_gpgkey_installed
+            - ensure_gpgcheck_never_disabled
+            - ensure_gpgcheck_local_packages
+            - ensure_gpgcheck_globally_activated
+            - security_patches_up_to_date
+            - dnf-automatic_security_updates_only
+        status: automated
+
+    -   id: 'mfa'
+        levels:
+            - base
+        title: 'Multi-factor authentication'
+        status: manual
+
+    -   id: 'restrict_admin'
+        levels:
+            - base
+        title: 'Restricting administrative privileges'
+        rules:
+            - accounts_no_uid_except_zero
+            - sudo_remove_nopasswd
+            - sudo_remove_no_authenticate
+            - sudo_require_authentication
+
+        status: partial
+
+    -   id: 'app_control'
+        levels:
+            - base
+        title: 'Application control'
+        rules:
+            - package_fapolicyd_installed
+            - service_fapolicyd_enabled
+        status: partial
+
+    -   id: 'restrict_macros'
+        levels:
+            - base
+        title: 'Restrict Microsoft Office macros'
+        status: not applicable
+
+    -   id: 'app_hardening'
+        levels:
+            - base
+        title: 'User application hardening'
+        status: pending
+
+    -   id: 'backups'
+        levels:
+            - base
+        title: 'Regular backups'
+        rules:
+            - package_rear_installed
+        status: partial
+
+    -   id: 'hardening'
+        levels:
+            - base
+        title: 'General hardening of {{{ full_name }}}'
+        rules:
+            - var_system_crypto_policy=default_nosha1
+            - configure_crypto_policy
+            - configure_ssh_crypto_policy
+            ### Secure access
```

Comment:
What do you think about creating sub-controls or separate controls for these subsections?

Reply:
There is no basis in the PDF for this, but we could do this. 

Would you be fine with the sub controls even though the PDF wouldn't support that?

Reply:
Aha OK then I think it would make problems for the automated reference assignement in future.

Reply:
so please keep it as it is now